Formula/mmseqs2.rb

```diff
@@ -0,0 +1,71 @@
+class Mmseqs2 < Formula
+  desc "Software suite for very fast protein sequence search and clustering"
+  homepage "https://mmseqs.org"
+  # tag "bioinformatics"
+  # doi "10.1038/nbt.3988"
+  url "https://github.com/soedinglab/MMseqs2.git", :using => :git, :tag => "1-c7a89", :revision => "c7a89b6aa6d94796783e94d49750b3e0882573aa"
```

Comment:
We prefer to build from release tarballs, like https://github.com/soedinglab/MMseqs2/archive/1-c7a89.tar.gz

Reply:
If I use the release tarball, MMseqs2 will not disable the git commit used to compile itself.

I can include a hack like this in the formula:
`inreplace "src/version/Version.cpp", /.+/m, "const char *version = \"#{version}\";"`
and fix it in some cleaner way for the next release. Would that be okay?